### PR TITLE
added volume information to material repr

### PR DESCRIPTION
**Pull request by shimwell**
Hi all, another tiny PR for your consideration

I'm doing a few depletion simulations and this requires adding volume information to each material.

It would be useful for me at least if the ```material.__repr__``` included the volume information.

Then when a material is printed it will display volume along with other handy things like name, density, temperature etc and I can easily see which of my materials still need volume adding

before this PR
```
 Material
	ID             =	8
	Name           =	mat_suspended_floor
	Temperature    =	None
	Density        =	7.82 [g/cm3]
	S(a,b) Tables  
	Nuclides       
	Fe54           =	0.057115528049999996 [ao]
```

after this PR
```
 Material
	ID             =	8
	Name           =	mat_suspended_floor
	Temperature    =	None
	Density        =	7.82 [g/cm3]
	Volume         =	1.2709345346346374e+26 [cm3]
	S(a,b) Tables  
	Nuclides       
	Fe54           =	0.057115528049999996 [ao]
```

### CODE REVIEW

**Comment by shimwell** [comment]
oh i see this has been proposed in #2368

closing in favor of #2368

File: openmc/material.py [hunk 1]
```diff
@@ -143,6 +143,9 @@ def __repr__(self):
         string += '{: <16}=\t{}'.format('\tDensity', self._density)
         string += f' [{self._density_units}]\n'
 
+        string += '{: <16}=\t{}'.format('\tVolume', self._volume)
+        string += ' [cm3]\n'
```
Comment (shimwell) [inline openmc/material.py:147]:
```suggestion
        string += ' [cm^3]\n'
```

**Comment by shimwell** [comment]
reopened as it looks like #2368 is not going ahead

File: openmc/material.py [hunk 2]
```diff
@@ -143,6 +143,9 @@ def __repr__(self):
         string += '{: <16}=\t{}'.format('\tDensity', self._density)
         string += f' [{self._density_units}]\n'
 
+        string += '{: <16}=\t{}'.format('\tVolume', self._volume)
+        string += ' [cm^3]\n'
```
Comment (paulromano) [inline openmc/material.py:147]:
```suggestion
        string += '{: <16}=\t{} [cm^3]\n'.format('\tVolume', self._volume)
```

**Review by paulromano — CHANGES_REQUESTED** [review]

**Review by paulromano — APPROVED** [review]

**Pull request merged**